yaml/com/ibm/License/Entry/LicenseEntry.interface.yaml

```diff
@@ -0,0 +1,67 @@
+description: >
+    Implement to represent a license which enables features on system
+    Customer installs license to unlock certain resources or features
+    Each object implements xyz.openbmc_project.License.Entry interface
+    which has Name and Status properties
+    and implements xyz.openbmc_project.Object.Delete interface.
+properties:
+
+    - name: Name
+      type: string
+      description: >
+         Name of license
+    - name: SerialNum
```

Comment:
Can we spell it out to match the existing interface SerialNumber

https://github.com/openbmc/phosphor-dbus-interfaces/blob/08baf48ad5f15774d393fbbf4e9479a0ef3e82d0/yaml/xyz/openbmc_project/Inventory/Decorator/Asset.interface.yaml#L8